InnerEye/ML/SSL/lightning_modules/simclr_module.py

```diff
@@ -57,6 +58,17 @@ def __init__(self, encoder_name: str, dataset_name: str, use_7x7_first_conv_in_r
     def forward(self, x: torch.Tensor) -> torch.Tensor:
         return self.encoder(x)
 
+    def training_step(self, batch: BatchType, batch_idx: int) -> torch.Tensor:
```

Comment:
Should the return type here be consistent with that in line 67?

Reply:
yes, thanks. Removed T altogether.